ui/app/services/keyboard.js

```diff
@@ -348,6 +348,13 @@ export default class KeyboardService extends Service {
       const shifted = event.getModifierState('Shift');
       if (type === 'press') {
         if (key === 'Shift') {
+          // if cmd or ctrl are pressed, don't show hints — this is likely a user trying to take a screenshot.
+          if (
+            event.getModifierState('Meta') ||
+            event.getModifierState('Control')
```

Comment:
who's `Control` for?  on my linux laptop, I just press `print sc` button (maybe with a `fn` modifier key, since it shares `F10` in my case)